p2p/transport/webrtc/stream.go

```diff
@@ -19,6 +21,9 @@ const (
 	// We can change this value in the SettingEngine before creating the peerconnection.
 	// https://github.com/pion/webrtc/blob/v3.1.49/sctptransport.go#L341
 	maxBufferedAmount = 2 * maxMessageSize
+	// maxTotalControlMessagesSize is the maximum total size of all control messages we will
+	// write on this stream
+	maxTotalControlMessagesSize = 500
```

Comment:
where does this number come from? 

giving the rationale or link to reference would be great!

Reply:
This should be 50. Added an explanation. Please comment if it needs elaborating.

Reply:
It is still `500` (not `50`) and I don't see the explanation